frontend/src/components/dashboard/components/courses-list/styles.module.scss

```diff
@@ -1,5 +1,6 @@
 .container {
   display: grid;
-  grid-template-columns: auto auto auto auto;
+  grid-template-columns: repeat(4, 360px);
```

Comment:
should be infinity with wrappring

Reply:
https://www.loom.com/share/f375864c45c749a7a4d9edf7b8068a42
Is this what you mean? It's now fully responsive + adaptive